gallery/visualtests/nxButton.js

```diff
@@ -115,7 +115,8 @@ describe('NxButton', function() {
     };
 
     describe('Secondary styled button', function() {
-      const secondarySelector = `${selector} .nx-btn:not(.nx-btn--primary):not(.nx-btn--tertiary):not(.nx-btn--error)`;
+      const secondarySelector = `${selector} .nx-btn:not`
+      + '(:is(.nx-btn--primary, .nx-btn--tertiary, .nx-btn--error, .nx-btn--small))';
```

Comment:
This is a really odd place to break the string into two lines.  I suggest breaking it on the space before `.nx-btn`.

Reply:
https://github.com/sonatype/sonatype-react-shared-components/pull/1220/commits/d7363ee2fb50ba3d6f4a2047de3470128540a8f3